test/runtime/Policies.go

```diff
@@ -669,6 +671,239 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
+		Expect(areEndpointsReady).To(BeTrue())
+
+		// Delete the pseudo-host IPs that we added to localhost after test
+		// finishes. Don't care about success; this is best-effort.
+		cleanup := func() {
+			_ = vm.Exec(fmt.Sprintf("sudo ip addr del dev lo %s/32", ipv4Host))
+			_ = vm.Exec(fmt.Sprintf("sudo ip addr del dev lo %s/128", ipv6Host))
+		}
+
+		defer cleanup()
+
+		By("Adding Pseudo-Host IPs to localhost")
+		res = vm.Exec(fmt.Sprintf("sudo ip addr add dev lo %s/32", ipv4Host))
+		res.ExpectSuccess("Unable to add %s to pseudo-host IP to localhost", ipv4Host)
+		res = vm.Exec(fmt.Sprintf("sudo ip addr add dev lo %s/128", ipv6Host))
+		res.ExpectSuccess("Unable to add %s to pseudo-host IP to localhost", ipv6Host)
+
+		By("Pinging host IPv4 from httpd2 (should NOT work due to default-deny PolicyEnforcement mode)")
+
+		res = vm.ContainerExec(helpers.Httpd2, helpers.Ping(ipv4Host))
+		res.ExpectFail("Unexpected success pinging host (%s) from %s: %s", ipv4Host, helpers.Httpd2, res.CombineOutput().String())
+
+		By(fmt.Sprintf("Importing L3 CIDR Policy for IPv4 Egress Allowing Egress to %s, %s from %s", ipv4OtherHost, ipv4OtherHost, httpd2Label))
+		script := fmt.Sprintf(`
+		[{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": 
+			[{
+				"toCIDR": [
+					"%s/24",
+					"%s/20"
+				]
+			}]
+		}]`, httpd2Label, ipv4OtherHost, ipv4OtherHost)
+		renderAndImportPolicy(script)
+
+		By(fmt.Sprintf("Pinging host IPv4 (%s) from %s (should work) because it is contained within CIDR %s", ipv4Host, helpers.Httpd2, ipv4OtherHost))
+		res = vm.ContainerExec(helpers.Httpd2, helpers.Ping(ipv4Host))
+		res.ExpectSuccess("Unexpected failure pinging host (%s) from %s: %s", ipv4Host, helpers.Httpd2, res.CombineOutput().String())
+
+		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
+
+		By("Pinging host IPv6 from httpd2 (should NOT work because we did not specify IPv6 CIDR of host as part of previously imported policy)")
+		res = vm.ContainerExec(helpers.Httpd2, helpers.Ping6(ipv6Host))
+		res.ExpectFail("Unexpected success pinging host (%s) from %s", ipv6Host, helpers.Httpd2)
+
+		By("Importing L3 CIDR Policy for IPv6 Egress")
+		script = fmt.Sprintf(`
+			[{
+		    "endpointSelector": {"matchLabels":{"%s":""}},
+		    "egress": [{
+			"toCIDR": [
+			    "%s"
+			]
+		    }]
+		}]`, httpd2Label, ipv6Host)
+		renderAndImportPolicy(script)
+
+		By(fmt.Sprintf("Pinging host IPv6 from httpd2 (should work because policy allows IPv6 CIDR %s)", ipv6Host))
+		res = vm.ContainerExec(helpers.Httpd2, helpers.Ping6(ipv6Host))
+		res.ExpectSuccess("Unexpected failure pinging host (%s) from %s: %s", ipv6Host, helpers.Httpd2, res.CombineOutput().String())
+
+		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
+
+		// This test case checks that ping works even without explicit CIDR policies
+		// imported.
+		By("Importing L3 Label-Based Policy Allowing traffic from httpd2 to httpd1")
+		script = fmt.Sprintf(`
+			[{
```

Comment:
If you can remove a tab here. Keep consistent as line 759

Reply:
Done